<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportOnPersonSetupDialog</class>
 <widget class="QDialog" name="ReportOnPersonSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblSpec">
     <property name="text">
      <string>Специальность</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbSpeciality">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>&amp;Врач</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbPerson">
     <item>
      <property name="text">
       <string>Врач</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>111</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblIdentifierType">
     <property name="text">
      <string>Тип идентификатора</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="cmbIdentifierType">
     <item>
      <property name="text">
       <string>Идентификатор клиента</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Идентификатор во внешней учётной системе</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportOnPersonSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportOnPersonSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
